src/main/java/core/basesyntax/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy;
+
+import java.util.Map;
+
+public class ReturnOperation implements FruitOperationStrategy {
+
+    @Override
+    public void execute(Map<String, Integer> fruitQuantity, String fruit, int quantity) {
+        fruitQuantity.put(fruit, fruitQuantity.get(fruit) + quantity);
```

Comment:
Consider checking if the fruit exists in the map before performing the addition to avoid potential `NullPointerException`. You might want to initialize the quantity to zero if the fruit is not present.